go-controller/observability-lib/ovsdb/model.go

```diff
@@ -0,0 +1,319 @@
+// Code generated by "libovsdb.modelgen"
+// Edited by npinaeva to only leave required tables.
```

Comment:
Editing a generated file can hide problems in the future (e.g: someone updates the schema and forgets to rebase your change and you are suddenly adding _all_ tables to your cache).

You can build your "smaller" db model map in the client code directly since `NewClientDBModel`  is public

Reply:
Thinking more about this, I don't expect a lot of records to live in the OVS database so monitoring the whole thing is probably acceptable.  Maybe we can just leave all tables here.

Reply:
It'll be a waste of resources IMHO. It's not only about memory (we need to put the entire OVSDB in the cache) but also processing time as every change in the OVSDB (e.g: interface statistics, port removal/creation, etc) will cause a notification to be received so the cache is updated. I don't think that is necessary at all when you can just easily move those lines to `nb_client.go`:

```go
	clientDBModel, err :=  model.NewClientDBModel("Open_vSwitch", map[string]model.Model{
		"Bridge":                    &Bridge{},
		"Flow_Sample_Collector_Set": &FlowSampleCollectorSet{},
	})

```

Reply:
OK, this is simple indeed.  +1 for this then.

Reply:
What I tried to avoid is adding all db files to `ovsdb` folder, while in reality I only need 2 tables.
I think it would be nice to have automation to generate models only for a subset of tables, but libovsdb can't do that yet.
I have added a manually created file `observ_model.go` and removed `model.go`.
The idea is that no one should ever touch that folder until we need new bridge or flow_sample_collector features (potentially never). And hopefully the absence of other model files makes it obvious that we don't use them.
If someone re-generates models from scratch, they will have a bunch of new files, and no changes on the indexed files (until bridge or flow_sample_collector_set changes).

Reply:
OK, this works for me.